Examples/Tests/collision/analysis_collision_2d.py

```diff
@@ -51,8 +52,18 @@
 b = -0.083851393560288
 
 last_fn = sys.argv[1]
+# Remove trailing '/' from file name, if necessary
 if (last_fn[-1] == "/"): last_fn = last_fn[:-1]
-fn_list = glob(last_fn[:-5] + "?????")
+# Find last iteration in file name, such as 'test_name_plt000001' (last_it = '000001')
+last_it = re.search('\d+', last_fn).group()
+# Find output prefix in file name, such as 'test_name_plt000001' (prefix = 'test_name_plt')
+prefix = last_fn[:-len(last_it)]
+# Loop over all files in current directory and save those whose name matches the pattern
+# prefix + arbitrary number ('\d+'), in order to collect all output files in fn_list.
+fn_list = []
+for fn in os.listdir('.'):
+    if re.match(prefix + '\d+', fn):
```

Comment:
Same maybe applies here, if the code ever starts writing files with related names plus an extension in the same directory